### PR TITLE
update staged deps config

### PR DESCRIPTION
**Pull request by pawelru**
test with https://github.com/openpharma/staged.dependencies/pull/182

### CODE REVIEW

**Review by Melkiades — COMMENTED** [review]
It seems not to work atm. It is still not directing staged.deps towards the subdir

**Comment by pawelru** [comment]
> It seems not to work atm. It is still not directing staged.deps towards the subdir

Have you tested this locally? CI is using old version. I think I can temporarily replace it with my feature branch. Let me check.

File: .github/workflows/check.yml [hunk 1]
```diff
@@ -22,7 +22,7 @@ jobs:
     uses: insightsengineering/r.pkg.template/.github/workflows/audit.yaml@main
   r-cmd:
    name: R CMD Check 🧬
-   uses: insightsengineering/r.pkg.template/.github/workflows/build-check-install.yaml@main
+   uses: insightsengineering/r.pkg.template/.github/workflows/build-check-install.yaml@support-subdir-v2
```
Comment (pawelru) [inline .github/workflows/check.yml:25]: need to revert back

**Comment by pawelru** [comment]
got success using `main` version of `staged.dependencies` package ✅

**Review by Melkiades — APPROVED** [review]
Once reverted, it is good to go for me! Thanks Pawel ;)

File: .github/workflows/check.yml (hunk 1 above)
Comment (pawelru) [inline .github/workflows/check.yml:25]:
```suggestion
   uses: insightsengineering/r.pkg.template/.github/workflows/build-check-install.yaml@main
```

**Pull request merged**